bin/relay/src/main.rs

```diff
@@ -0,0 +1,75 @@
+//! # Odyssey Relay
+//!
+//! TBD
+
+use alloy_provider::{network::EthereumWallet, Provider, ProviderBuilder};
+use alloy_rpc_client::RpcClient;
+use alloy_signer_local::PrivateKeySigner;
+use clap::Parser;
+use eyre::Context;
+use jsonrpsee::server::Server;
+use odyssey_wallet::{AlloyNode, OdysseyWallet, OdysseyWalletApiServer};
+use reth_tracing::Tracer;
+use std::net::{IpAddr, Ipv4Addr};
+use tracing::info;
+use url::Url;
+
+/// The Odyssey relayer service sponsors transactions for EIP-7702 accounts.
+#[derive(Debug, Parser)]
+#[command(author, about = "Relay", long_about = None)]
+struct Args {
```

Comment:
maybe should be moved into the lib and be importable in odyssey binary with clap flatten under `relayer.http.address`, `--relayer.signer` etc.?